pkg/app/src/components/Account/generalLayout.tsx

```diff
@@ -10,9 +10,9 @@ import SectionsLayout from './sectionsLayout'
 
 interface AccountPageGeneralLayoutProps {
 	accountState: AccountState
+	param: AccountTabs
 }
-const AccountPageGeneralLayout: FC<AccountPageGeneralLayoutProps> = ({ accountState }) => {
-	const [currentTab, setCurrentTab] = useState<AccountTabs>(AccountTabs.OVERVIEW)
+const AccountPageGeneralLayout: FC<AccountPageGeneralLayoutProps> = ({ accountState, param }) => {
```

Comment:
1. Do not write FC components.
2. Rename AccountPageGeneralLayoutProps  to ComponentProps
3. No default export.
4. Do not pass AccountState or Values you get from a hook as parameter (avoid useless state management)